### PR TITLE
Home work

### PR DESCRIPTION
**Pull request by Apoerin**
image

### CODE REVIEW

**Comment by Apoerin** [comment]
image

File: HW2/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#folder idea
```
Comment (spolischook) [inline HW2/.gitignore:1]: Use only one gitignore in the root of project

File: HW2/form.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+</head>
+<body>
+<h1>Profile edit page</h1>
+<form>
+    <fieldset>
+        <legend>Basic info</legend>
+            <ul>
+                <li>
+                    <label for="iduser">ID</label>
+                    <input type="text" id="iduser" placeholder="725">
```
Comment (spolischook) [inline HW2/form.html]: Use it readonly or disabled

File: HW2/form.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                    <label for="firstName">First Name</label>
+                    <input type="text" id="firstName">
+                </li>
+                <li>
+                    <label for="lastName">LastName</label>
+                    <input type="text" id="LastName">
+                </li>
+                <li>
+                    <label for="password">Password</label>
+                    <input type="Password" id="password">
```
Comment (spolischook) [inline HW2/form.html]:
```suggestion
                    <input type="password" id="password">
```

File: HW2/form.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                    <input type="tel" id="phone" placeholder="380">
+                </li>
+                <li>
+                    <label for="email">Email</label>
+                    <input type="email" id="email" placeholder="myemail@placeholder.com">
+                </li>
+                <li>
+                    <label>
+                        <input type="checkbox">Keep my email adress private</label><br>
+                    We'll remove your public profile email and use 1978331+username@user.noreply.localhost
```
Comment (spolischook) [inline HW2/form.html]: This text is hang, wrap it

File: HW2/form.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                    <select name="country">
+                        <option>- Choose your country -</option>
+                        <option>Ukraine</option>
+                        <option>USA</option>
+                        <option>Thailand</option>
+                        <option>Spain</option>
+                    </select>
+                </li>
+                <li>
+                    <input name="surprise" type="hidden">
```
Comment (spolischook) [inline HW2/form.html]: value is missing here

File: HW2/form.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                <li>
+                    <label>Development experience</label><br>
+                        <input type="radio">Learning to Program<br>
+                        <input type="radio" checked>Junior Developer<br>
+                        <input type="radio">Mid-Level Developer<br>
+                        <input type="radio">Senior Developer<br>
+                </li>
+                <li>
+                    <label>Country</label>
+                    <select name="country">
```
Comment (spolischook) [inline HW2/form.html]: value is missing for options

File: HW2/form.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+                </li>
+            </ul>
+        </fieldset>
+    <br>
+    <input type="reset" value="Cancel">
+    <input type="submit" value="Submit">
+</form>
+
+</body>
+</html>
```
Comment (spolischook) [inline HW2/form.html]: settings_025

**Review by spolischook — CHANGES_REQUESTED** [review]
Missed name attribute for form fields

File: HW2/form.html [hunk 8]
```diff
@@ -67,28 +66,28 @@ <h1>Profile edit page</h1>
...
                 </li>
                 <li>
                     <label>Country</label>
                     <select name="country">
-                        <option>- Choose your country -</option>
-                        <option>Ukraine</option>
-                        <option>USA</option>
-                        <option>Thailand</option>
-                        <option>Spain</option>
+                        <option value="choose">- Choose your country -</option>
```
Comment (spolischook) [inline HW2/form.html]: :sun_with_face:

**Pull request merged**